src/constants.js

```diff
@@ -22,6 +22,8 @@ export const CLAIM_ADMIN_USER_TYPE = "CLAIM_ADMIN";
 export const CLAIM_ADMIN_IS_SYSTEM = 256;
 export const MODULE_NAME = "user";
 
+export const EMAIL_REGEX_PATTERN = /^[^\s@]+@[^\s@]+\.[^\s@]+$/;
```

Comment:
Can you fetch the regex from BE (exposed configuration ? )

Reply:
https://github.com/openimis/openimis-be-core_py/pull/164/files

Reply:
Do you mean that we should expose an endpoint and fetch if the format is valid or not? @delcroip

Reply:
no just the regex for email or at least copy it from be